### ISSUE TITLE
Basic description of TensorBoard plots

### ISSUE THREAD

**Issue by obo** [ufal/neuralmonkey#269]
Could someone add a basic description and "standard" pictures of the plots that NM records in TensorBoard? So that (mainly newcomers) would know what they should see and what they should not see.

**Comment by jindrahelcl** [comment]
Do you have an example of such newcomer? I think how the plots should look like is more of a NMT-research-related problem - how is that specific to neuralmonkey? (Just clarifying what do you want to have)

**Comment by obo** [comment]
Yes, it is NMT-research-related problem and NM should be *the* :-) entry point to NMT. So it should help people also with basic NMT things, wherever it easily shows them.

----- Original Message -----
> From: "Jindra Helcl" <notifications@github.com>
> To: "ufal/neuralmonkey" <neuralmonkey@noreply.github.com>
> Cc: "Ondrej Bojar" <bojar@ufal.mff.cuni.cz>, "Author" <author@noreply.github.com>
> Sent: Monday, 23 January, 2017 00:03:17
> Subject: Re: [ufal/neuralmonkey] Basic description of TensorBoard plots (#269)

> Do you have an example of such newcomer? I think how the plots should look like
> is more of a NMT-research-related problem - how is that specific to
> neuralmonkey? (Just clarifying what do you want to have)
> 
> --
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/ufal/neuralmonkey/issues/269#issuecomment-274367494

-- 
Ondrej Bojar (mailto:obo@cuni.cz / bojar@ufal.mff.cuni.cz)
http://www.cuni.cz/~obo

**Comment by jindrahelcl** [comment]
It's definitely a nice idea... But I don't want to do it.. :-) I think its far beyond the scope of the software (at this moment). We could do something like this later - I'm thinking lab sessions at MT marathon - and then include the materials in documentation, or just refer to them from it.

If there is a free pair of hands ready to make contributions, I'd prefer to do something else.
